pkg/cmdutil/repo_override.go

```diff
@@ -11,6 +11,11 @@ func EnableRepoOverride(cmd *cobra.Command, f *Factory) {
 	cmd.PersistentFlags().StringP("repo", "R", "", "Select another repository using the `[HOST/]OWNER/REPO` format")
 
 	cmd.PersistentPreRun = func(cmd *cobra.Command, args []string) {
+		// Execute rootCmd's PersistentPreRun hook
+		if rootPPreRun := cmd.Root().PersistentPreRun; rootPPreRun != nil {
+			rootPPreRun(cmd, args)
+		}
```

Comment:
This is needed in order to execute rootCmd's PersistentPreRun hook on all sub-commands that use `EnableRepoOverride` to set their own hook, see https://github.com/spf13/cobra/issues/1198.